custom_components/e3dc_rscp/coordinator.py

```diff
@@ -136,6 +138,29 @@ def wallboxes(self, value: list[dict[str, str | int]]) -> None:
         """Set the list of wallboxes."""
         self._wallboxes = value
 
+    # Setter for individual wallbox values
+    def setWallboxValue(self, index: int, key: str, value: Any) -> None:
+        """Set the value for a specific key in a wallbox identified by its index."""
+        for wallbox in self._wallboxes:
+            if wallbox['index'] == index:
+                wallbox[key] = value
```

Comment:
@bullitt186 do we want this as generic function like this? I normally prefer typed specific accessors, they make for cleaner code ad catch errors early.

Reply:
Good point. And it is not being used in any situation at all (and probably never will be, as wallboxes are only added by the coordinator itself.)
I remove it.